sw/device/lib/dif/dif_edn.h

```diff
@@ -42,6 +42,22 @@ enum {
   kDifEntropySeedMaterialMaxWordLen = 12,
 };
 
+enum {
+  /**
+   * Maximum seed material number of uint32_t words supported in CSRNG
+   * instantiate and seed commands.
+   */
+  kDifEntropySeedMaterialMaxGlen = 4095,
```

Comment:
This documentation is identical to both of the other enums, I think. (And the values are different!)

I think the comments need a bit of rewriting?

Reply:
You're right, thanks @rswarbrick!